src/main/webapp/app/resources/styles/sass/repositoryViewContext/_metadataSection.scss

```diff
@@ -15,4 +15,16 @@
       }
     }
   }
+
+  @media (max-width: 767px) {
```

Comment:
The PR looks good to me, but I'd prefer it if we could have this breakpoint at a standard size instead of one pixel under.